.cirrus.yml

```diff
@@ -95,11 +94,33 @@ gating_task:
         - '/usr/local/bin/entrypoint.sh lint'
 
 
+master_build_task:
+    depends_on:
+        - "gating"
+
+    gce_instance:
+        image_project: "libpod-218412"
+        zone: "us-central1-a"  # Required by Cirrus for the time being
+        cpu: 2
+        memory: "4Gb"
+        disk: 40
+        matrix:
+            image_name: "ubuntu-18-libpod-0c954a67"
+            image_name: "fedora-29-libpod-0c954a67"
+
+    timeout_in: 20m
+
```

Comment:
[Found docs that will make this even easier](https://cirrus-ci.org/guide/writing-tasks/#custom-clone-command) and no need to do git-stuff in `Makefile`:

```yaml
    ...
    clone_script: 
        - git clone --branch $CIRRUS_BASE_BRANCH https://github.com/${CIRRUS_REPO_FULL_NAME}.git $CIRRUS_WORKING_DIR
    script:
        ...
```